scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -158,46 +158,61 @@ def fetch_releases(client: httpx.Client, repo: str) -> list[Release]:
                 id=int(item["id"]),
                 tag_name=item["tag_name"],
-                name=item["name"],
+                name=item.get("name", item["tag_name"]),
                 prerelease=item["prerelease"],
-                body=body if body and body.strip() else None,
-                published_at=item["published_at"],
+                body=item.get("body"),
+                published_at=item.get("published_at", item.get("created_at", "")),
                 assets=item.get("assets", []),
             )
         )
```

Ruff (0.15.15)

[warning] Use a list comprehension to create a transformed list

(PERF401)